app/controllers/concerns/cache_concern.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module CacheConcern
+  extend ActiveSupport::Concern
+
+  def render_with_cache(**options)
+    raise ArgumentError, 'only JSON render calls are supported' unless options[:json].present? || block_given?
+
+    key        = options.delete(:key) || request.url
```

Comment:
This is the part I don't really like--using `request.url` allows cache-busting too easily, and we don't differentiate between bare/full actor with this. My criteria however is to keep `render_with_cache` calls as minimal as possible instead of manually supplying a long key definition...

Reply:
That is definitely an issue, at least wrt. the bare/full actor thing.

Reply:
Could use controller+action+cache_key of the object, if the object responds to cache_key

Reply:
That still wouldn't help, would it? The bare/full actor thing is a difference in the `fields` option, not the actor itself. You could pass the key explicitly for that case.